protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -221,7 +221,12 @@ String _unCamelCase(String name) {
 }
 
 class MapFieldInfo<K, V> extends FieldInfo<PbMap<K, V>?> {
+  /// Key type of the map. Per proto2 and proto3 specs, this needs to be an
+  /// integer type or `string` -- i.e. no floats or `bytes`.
   final int? keyFieldType;
+
+  /// Value type of the map. Per proto2 and proto3 specs, this can be any type
+  /// other than `map`.
```

Comment:
I don't think it can be repeated?

Reply:
Fixed. I think the spec considers repeated as a modifier or attribute rather than type, but this is not clear in the current `field_type.dart` where everything is a bit, multiple type bits can be set etc. It will be clear with #605 so I will update this comment in #605 to say something like "this can be any type other than `map`, and the type cannot be repeated".